cmd/spr/main.go

```diff
@@ -136,11 +136,19 @@ VERSION: {{.Version}}
 				Aliases: []string{"u", "up"},
 				Usage:   "Update and create pull requests for updated commits in the stack",
 				Action: func(c *cli.Context) error {
+					if c.IsSet("reviewer") {
+						stackedpr.Reviewers = append(stackedpr.Reviewers, c.StringSlice("reviewer")...)
```

Comment:
pass the reviewers to the UpdatePullRequest function as an argument instead of assigning it here. 
have a look at the upto flag in the MergePullRequests for example